.github/workflows/ossar-analysis.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow integrates a collection of open source static analysis tools
+# with GitHub code scanning. For documentation, or to provide feedback, visit
+# https://github.com/github/ossar-action
+name: OSSAR
+
+on:
+  push:
+  pull_request:
```

Comment:
Should these keys be set?

Reply:
This is the default workflow for the tool. I believe this sets it to run on all pushes and PRs.

Reply:
Wonder if we should have it for pushes and PRs to master only, as we do for the other security workflows?

Reply:
I'd leave it on the default settings first and change later if needed.